modules/gitops-release-notes-1-11-4.adoc

```diff
@@ -0,0 +1,45 @@
+
+Issued: 2024-04-08
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:130841-02[RHSA-2024:130841-02]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-11-4_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE and EKS clusters because this binary was not available in the {gitops-shortname} container. This update fixes the issue by adding the `argocd-k8s-auth` binary in the {gitops-shortname} container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, the attempt to connect to Azure DevOps with Argo CD would result in an error due to the deprecation of the `rsa-ssh` host key algorithm by the Azure DevOps Repository service. This update fixes the issue by providing support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to log in to the Argo CD web console due to an error indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors when the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+* Before this update, users were unable to log in to the Argo CD web console if the availability of `openshift-gitops` route was delayed while the {gitops-title} Operator processed an Argo CD custom resource instance. An error message was displayed indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors. link:https://issues.redhat.com/browse/GITOPS-3736[GITOPS-3736]
+
+* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases. This update fixes the issue because operator backed resources with the correct versions are included in the `ClusterServiceVersion` manifest file.
```

Comment:
Same comment as in the other PR:
```suggestion
* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases. This update fixes the issue because Operator-backed resources with the correct versions are included in the `ClusterServiceVersion` manifest file.
```